viewer/packages/data-providers/src/image-360-data-providers/Cdf360ImageEventProvider.ts

```diff
@@ -61,20 +70,43 @@ export class Cdf360ImageEventProvider implements Image360Provider<Metadata> {
     image360FaceDescriptors: Image360FileDescriptor[],
     abortSignal?: AbortSignal
   ): Promise<Image360Face[]> {
-    const fileIds = image360FaceDescriptors.map(image360FaceDescriptor => {
+    const fullResFileBuffers = await this.getFileBuffers(this.getFileIds(image360FaceDescriptors), abortSignal);
+    return this.createFaces(image360FaceDescriptors, fullResFileBuffers);
+  }
+
+  public async getLowResolution360ImageFiles(
+    image360FaceDescriptors: Image360FileDescriptor[]
+  ): Promise<Image360Face[]> {
+    validateFileFormat();
+    const lowResFileBuffers = await this.getIconBuffers(this.getFileIds(image360FaceDescriptors));
+    return this.createFaces(image360FaceDescriptors, lowResFileBuffers);
+
+    // Astrid-kg: This function can possibly be removed once
+    // https://github.com/cognitedata/ark/issues/6459 has been deployed.
+    function validateFileFormat() {
+      const validFormats = ['image/jpeg', 'image/png'];
```

Comment:
Following the link, it seems like this issue has already been solved?
https://github.com/cognitedata/ark/issues/6459

Reply:
I made a task to follow this up: https://cognitedata.atlassian.net/browse/REV-820